mlir/lib/Dialect/Rock/Generator/ConvGenerator.cpp

```diff
@@ -547,6 +548,13 @@ LogicalResult ConvGenerator::parseConvConfig(OpBuilder &builder,
   config.chipFeatures = splitter.getFeaturesForBackend();
   config.triple = splitter.getTriple().str();
 
+  FailureOr<amdgpu::Chipset> maybeChipset = amdgpu::Chipset::parse(config.chip);
```

Comment:
Use the chipset to choose the appropriate 8-bit float types for the architecture.  Here in parseConvConfig(), we won't see the --force-f8-types option because that's in rocmlir-gen.cpp, and we do have a --arch option and use the architecture that's given.